unicodetools/src/main/java/org/unicode/props/UcdPropertyValues.java

```diff
@@ -1890,6 +1890,39 @@ public static Line_Break_Values forName(String name) {
         }
     }
 
+    // Link_Bracket
+    public enum Link_Term_Values implements Named {
```

Comment:
Weird: These two lines don't go together.

Not a regression here because I see that the next section has the same problem:
```
    // Lowercase_Mapping
    public enum Math_Class_Values implements Named {
```